### DISCUSSION TITLE
How to configure http2 in nuxt3?

### DISCUSSION THREAD

**Discussion by EdivenH** [nuxt/nuxt#25927] [Questions] [answered]
nginx has enabled http2, but the access page protocol is still http1.1. Does nuxt also need to configure something to take effect?

I didn't see the relevant configuration in the documentation.

Can anyone help me with the answer? Thank you very much!!!

**Comment by TheAlexLichter** [comment, marked as answer]
It is enough if your reverse proxy is configured to serve http2 ☺️

**Reply by EdivenH** [reply]
Ok, thank you.
I found the problem
